modular_zubbers/code/modules/cargo/bounties/prisoner.dm

```diff
@@ -7,7 +7,7 @@
 
 /datum/bounty/item/prisoner/baseball_bat
 	name = "Baseball Bat"
-	description = "Baseball fever is going on at CentCom! Be a dear and ship them some baseball bats, so that management can live out their childhood dream."
+	description = "Security wants more bats, sell them... Don't use them unless you want to be the new league ball."
```

Comment:
The description for this one earlier was fine, this one is simply more unclear as it now specifies simply bats, when it used to specifically call out baseball bats

Reply:
This one was more thematic for prisoners, wasn't so much for clarity. Just didn't want an exact copy-paste of the assistant one.

Reply:
Maybe then specify that security wants more baseball bats, instead of shortening it to bats

Reply:
That's fair, I'll do that now.